backend/game_server/lib/cell.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Cell do
+	import Entity
+	alias Helpers.Physic, as: Physic
+	alias Helpers.Vector, as: Vector
+
+	def born(position,velocity) do
+		%Entity{} 
+			|> set_attribute(:position, position)
+			|> set_attribute(:velocity, velocity)
+			|> set_attribute(:mass, 5)
+			|> give_ability(:move, Cell)
+			|> give_ability(:eat, Cell)
+	end
+	def born(position), do: born(position, %Vector{})
+	def born(), do: born(%Vector{}, %Vector{})
+
+	def move(cell,t) do
+		new_position = Physic.move(get_attribute(cell,:position),get_attribute(cell,:velocity),t)
+		set_attribute(cell,:position, new_position)
```

Comment:
please, normalize function call and define white spaces. I believe you should always use space after argument post-comma. it's apply to few places in this PR code.